src/org/labkey/test/components/domain/DomainFieldRow.java

```diff
@@ -380,7 +380,7 @@ public DomainFieldRow setLookup(FieldDefinition.LookupInfo lookupInfo)
         String tableType = lookupInfo.getTableType();
         if (tableType == null)
             throw new IllegalArgumentException("No lookup type specified for " + lookupInfo.getTable());
-        setFromTargetTable(lookupInfo.getTable() + " (" + tableType + ")");
+        setFromTargetTable(lookupInfo.getTable() + " (" + tableType.toUpperCase() + ")");
```

Comment:
@labkey-ankurj on second thought I don't think toUpperCase() is what we want here. I just saw the TeamCity failure fo rFileContentUploadTest.testFileBrowser on this feature branch. We want to convert tableType from lower case to camel case (i.e. "string" -> "String") here.

Reply:
done